web-admin/src/features/projects/status/ProjectResourcesTable.svelte

```diff
@@ -96,13 +168,23 @@
       accessorKey: "actions",
       header: "",
       cell: ({ row }) => {
-        if (!isReconciling) {
+        // Only hide actions for reconciling rows
```

Comment:
UX: this is fine but, going forward, rather than _hiding_ elements like this, preferably we:
- _Show and disable_ the element(s)
- Include an explanation for _why_ the element(s) are disabled